Remotion/Web/Development.WebTesting.IntegrationTests/MultiWindowTest.cs

```diff
@@ -27,6 +28,20 @@ namespace Remotion.Web.Development.WebTesting.IntegrationTests
   [TestFixture]
   public class MultiWindowTest : IntegrationTest
   {
+    [Test]
+    public void TestMultiWindowResizeBrowserContent ()
+    {
+      var home = Start();
+      var newSize = new Size (600, 600);
+      var loadWindowFunctionInNewWindowButton = home.WebButtons().GetByID ("LoadWindowFunctionInNewWindow");
+      var window1 = loadWindowFunctionInNewWindowButton.Click().ExpectNewPopupWindow<WxePageObject> ("MyWindow");
+      var window2 = loadWindowFunctionInNewWindowButton.Click().ExpectNewPopupWindow<WxePageObject> ("MyWindow");

```

Comment:
We could extract the window title into a constant variable to emphasize it's the same one twice